notebooks/nightly_report/nightly_report_aos_eas.ipynb

```diff
@@ -1,4 +1,4 @@
~{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #282.    
</pre><p>It would be great to add an x-axis that displays the corresponding seqNum. That would help connect information contained here and that in the other reports</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/ts_aos_analysis/pull/53/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>